dvc/cli.py

```diff
@@ -117,6 +117,37 @@ def parse_args(self, args=None, namespace=None):
             self.error(msg % " ".join(argv), getattr(args, "func", None))
         return args
 
+    def parse_known_args(self, args=None, namespace=None):
+        self.add_common_args()
+
+        # TODO: temp, all subparser should be called with add_help=False
+        if self.prog in ("dvc", "dvc add", "dvc init"):
```

Comment:
Could you explain this line? Are we going to have to add each command to this list?

Reply:
nope, see my `TODO` comment, I haven't finished the implementation. If this solution works, I need go through all `add_parser` methods and remove `parent_parser` and add `add_help=False` (I wrote the same in PR description). After full cleanup, I can remove this block at all - this why it says `temp` :).